src/status_im/ui/screens/chat/views.cljs

```diff
@@ -423,6 +423,39 @@
        :inverted                     (when platform/ios? true)
        :style                        (when platform/android? {:scaleY -1})})]))
 
+(def show-floating-scroll-down-button (reagent/atom false))
+(def threshold-percentage-to-show-floating-scroll-down-button 75)
+
+(defn scroll-to-bottom []
+  (when @messages-list-ref
```

Comment:
you have to use `^js` for js values, this could be replaced by

`(some-> ^js @messages-list-ref (.scrollToOffset #js {:y 0 :animated true}))`